### ISSUE TITLE
Prompt for administrator login when opening user management 

### ISSUE THREAD

**Issue by TobiasGrether** [fsr5-fhaachen/portals#608]
Currently, when opening user management as an Administrator without having previously entered your administrator password in the tutor overview, all requests for the user list will fail because your session is not authenticated with the administrator password yet.

Really it should ask for your password by default when you haven't yet provided it, just as in the tutor view.

**Comment by simonostendorf** [comment]
I think this is a feature request and not a bug, because the current architecture "password promt only on dashboard page" works :)

**Comment by TobiasGrether** [comment]
> I think this is a feature request and not a bug, because the current architecture "password promt only on dashboard page" works :)

True, but at the same time this feature missing breaks the administrator user list by default. So while the feature itself works as expected it causes an issue in other places
